main/integration-tests/__test__/api-tests/aws-accounts/get-aws-accounts.test.js

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('List AWS accounts scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('List AWS accounts', () => {
+    it('should fail if user is inactive', async () => {
+      const admin2Session = await setup.createAdminSession();
+      await adminSession.resources.users.deactivateUser(admin2Session.user);
+
+      await expect(admin2Session.resources.awsAccounts.get()).rejects.toMatchObject({
+        code: errorCode.http.code.unauthorized,
+      });
+    });
+
+    it('should return empty AWS Accounts list for internal guest', async () => {
+      const guestSession = await setup.createUserSession({ userRole: 'internal-guest', projectId: [] });
+      await expect(guestSession.resources.awsAccounts.get()).resolves.toStrictEqual([]);
+    });
+
+    it('should fail if external guest attempts to get AWS Accounts list', async () => {
```

Comment:
Would you be willing to update the description to be consistent with the previous test description?  The api doe not actually fail, (there is no error code), the api returns empty list.  So the description can be 'should return empty AWS accounts list for external guest'.   Outside of this test, we might want to leave a comment in the spreadsheet so that this API should return unauthorized.